rbd/watchers_mimic.go

```diff
@@ -84,15 +87,15 @@ var watchCallbacks = callbacks.New()
 // callback.
 type WatchCallback func(interface{})
 
-type watchInstance struct {
+type watchCallbackItem struct {
```

Comment:
What I find even better, inspired by the C++ code, is `watchCallbackCtx`.

Reply:
Sure. Done.